test/accelerators/NNPA/backend/CMakeLists.txt

```diff
@@ -96,12 +96,15 @@ set(NNPA_TEST_LIST
     # ==ADDITIONAL_PARAGRAPH== NNPA has hardware limitations in dimension index size and tensor size, which are described in [NNPALimit.h](../src/Accelerators/NNPA/Conversion/ONNXToZHigh/NNPALimit.h). They are enough large for normal use cases, but if your model exceeds the limitations, CPU is used instead of NNPA.
 
     # ==OP== Add
+    # ==MIN== 14
     # ==LIM== - Shape of input tensors must be the same since broadcasting is not supported.<br>- Input tensors must have static dimensions.
     # Scalar tensor not supported.
     test_add_cpu,zdnn_add
     # test_add_bcast_cpu # bcast not supported
 
     # ==OP== AveragePool
+    # ==MAX== 19
```

Comment:
Please see the `onnx-mlir/utils/documentOps.py`. If the MAX or MIN are not defined that means it is defaulted to the values Alex hardcoded in the script. It should be lines 24 and 25 respectively.

Reply:
That makes sense. If there's a "true" max, that would show up here. If the max is just "latest currently available" then there shouldn't be a MAX value here and the script uses the default. So I'm thinking there shouldn't be a MAX value here because 19 is the latest and we support 19. (Unless by MAX, it means we _don't_ support AveragePool 19?)